src/styles/reset.scss

```diff
@@ -0,0 +1,12 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body{
+  margin: 0;
+  padding: 0;
```

Comment:
I would rather write like this:
```
* {
  box-sizing: border-box;
}

html {
  font-family: "Roboto", sans-serif;
  font-size: 12px;
}
```

and create file reset.scss
```
body,
p {
  margin: 0;
  padding: 0;
}
```

so now you can delete styles `margin: 0; padding: 0;` which you often use in card.scss and on `body`